pkg/config/azure.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"encoding/base64"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+)
+
+func (g *simpleGenerator) overrideAzureDefaults(cs *api.OpenShiftManagedCluster) {
```

Comment:
Should we be doing this? Or where do we want to overrides?

Reply:
discussion points - worth having default values for the images; probably worth rolling this into images.go